kingpin/actors/rightscale/server_array.py

```diff
@@ -742,3 +803,33 @@ def _execute(self):
             self.log.info('Completed %s tasks.' % count)
 
         raise gen.Return()
+
+    @gen.coroutine
+    def _execute(self):
+        # First things first, login to RightScale asynchronously to
```

Comment:
I added a docstring -- but its separate from this comment, which should remain a comment because it only relates to line 809.